src/main/java/com/bookingApp/bookingApp/model/Movie.java

```diff
@@ -0,0 +1,29 @@
+package com.bookingApp.bookingApp.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Entity
+@NoArgsConstructor
+@Data
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @NotNull
+    private String title;
+
+    @OneToMany(mappedBy = "movie")
+    @JsonIgnoreProperties("movie")
```

Comment:
jaka jest wada jednego obiektu zarówno do jsona jak i do bycia entity?

Reply:
przy wyswietlaniu zignoruje to pole i moze nie polaczyc tabel?